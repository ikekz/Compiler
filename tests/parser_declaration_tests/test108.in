procedure p(a:);
begin
end;
begin
end.