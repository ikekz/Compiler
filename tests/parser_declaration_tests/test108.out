(1,15) Error: Illegal type